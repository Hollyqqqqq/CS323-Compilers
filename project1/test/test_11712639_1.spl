int a, b, c
int
int func(){
    char s = '\xaa'
    a = 0
    return a
}