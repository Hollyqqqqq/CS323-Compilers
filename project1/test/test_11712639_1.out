Error type B at Line 2: Missing semicolon ';'
Error type B at Line 3: Missing semicolon ';'
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 6: Missing semicolon ';'
Error type B at Line 7: Missing semicolon ';'
